model_info/idwiki.reverted.md

```diff
@@ -15,67 +15,67 @@ Model Information:
 	 - python_implementation: 'CPython'
 	 - python_revision: ''
 	 - python_version: '3.5.3'
-	 - release: '4.9.0-6-amd64'
+	 - release: '4.9.0-8-amd64'
 	
 	Statistics:
-	counts (n=98898):
+	counts (n=98654):
 		label        n         ~True    ~False
 		-------  -----  ---  -------  --------
-		True       959  -->      405       554
-		False    97939  -->     1757     96182
+		True      2580  -->     1680       900
```

Comment:
Do you know what happened here?  This is a big change in the trusted labels, but I don't think the data set has updated beyond normal wear and tear?  Is this a bug in the merge logic?

It damaged model health, so this is worth investigation.

Reply:
It seems that our autolabeling script got better at catching potentially damaging edits.  This tends to have a minor effect on our performance statistics.  I wouldn't consider this a big change in model health as many of the statistics are sensitive to rates in the target class.